bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -149,6 +149,11 @@ spec:
             {{- end }}
               containerPort: {{ .Values.master.webPort }}
               protocol: TCP
+            {{- if .Values.security.ssl.enabled }}
+            - name: https
+              containerPort: {{ .Values.master.webPortHttps }}
+              protocol: TCP
+            {{- end }}
```

Comment:
Same issue as the Service. This causes `https` port to be duplicated.

In this case I would suggest:
```
            {{- if not .Values.security.ssl.enabled }}
            - name: http
              containerPort: {{ .Values.master.webPort }}
              protocol: TCP
            {{- else }}
            - name: https
              containerPort: {{ .Values.master.webPortHttps }}
              protocol: TCP
            {{- end }}
```

Reply:
Changed in d228c74ef